docs/ARCHITECTURE.md

```diff
@@ -0,0 +1,52 @@
+
+# Architecture
+
+## Diagram
+
+(This diagram is generated by [mermaid](https://mermaid-js.github.io/mermaid/#/./flowchart?id=flowcharts-basic-syntax))
+
+```mermaid
+graph LR
+
+    subgraph GMS Access
+        GMS_APIs((GMS APIs))
+        GMS_Dumps[(GMS Data Dumps)]
+    end
+    subgraph Client Applications
+        Scraper[[GMS Scrapers]]
+        Ingestor[[GMS Ingestors]]
+        UX{{User Interfaces}}
+        GMS[Integrated GMSs]
+        Infomediaries[[Infomediaries]]
+    end
+    subgraph PDC Backend
+        API((API))
+        Database[(Data Store)]
+    end
+    subgraph External Services
+        GMS_APIs((GMS APIs))
+        GMS_Dumps[(GMS Data Dumps)]
+        Auth((KeyCloak))
+        Email((MailGun))
+    end
+
+    GMS---API
+    Infomediaries---API
+    Scraper---GMS_APIs
+    Scraper---API
+    Ingestor---API
```

Comment:
I have seen it spelled both ways, "ingester" and "ingestor" but cannot find any definitive answers online :shrug: so I suppose it's all good. I will have a hard time spelling it with the "o" so forgive me if I commit something conflicting.

Reply:
I spent so much time looking into this for another project lol and I reached no meaningful conclusions.

I went with the `o` variety on that project because there were also "generators" and... ugh, language.

I'll swap to the `e` here to make life easier!